examples/fluids/navierstokes.c

```diff
@@ -107,19 +107,21 @@ int main(int argc, char **argv) {
   CeedInit(app_ctx->ceed_resource, &ceed);
```

Comment:
Need to check for a clean return here, then this looks good to merge to me

Reply:
Done.